st2client/st2client/shell.py

```diff
@@ -78,6 +78,11 @@
     'debug': ['cli', 'debug']
 }
 
+# A list of command classes for which automatic authentication should be skipped.
+SKIP_AUTH_CLASSES = [
+    'TokenCreateCommand'
```

Comment:
Can you use an import so that name changes impact are immediate and obvious. 

e.g. if TokenCreateCommand is renamed we will see a very visible regression.